### PR TITLE
[GHA] Split nightly scope

### PR DESCRIPTION
**Pull request by mvafin**
### Details:
 - *Move pytorch transformation tests*
 - *Split nightly scope*

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: tests/model_hub_tests/models_hub_common/test_convert_model.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 import numpy as np
 # noinspection PyUnresolvedReferences
-import openvino_tokenizers  # do not delete, needed for text models
+#import openvino_tokenizers  # do not delete, needed for text models
```
Comment (mvafin) [inline tests/model_hub_tests/models_hub_common/test_convert_model.py:7]:
```suggestion
import openvino_tokenizers  # do not delete, needed for text models
```

File: .github/workflows/job_pytorch_models_tests.yml [hunk 2]
```diff
@@ -123,48 +131,47 @@ jobs:
...
           USE_SYSTEM_CACHE: False
           OP_REPORT_FILE: ${{ env.INSTALL_TEST_DIR }}/TEST-torch_unsupported_ops.log
 
       - name: PagedAttention Test
-        if: always()
+        if: ${{ inputs.model_scope == 'precommit' }}
         run: |
           export PYTHONPATH=${MODEL_HUB_TESTS_INSTALL_DIR}:$PYTHONPATH
-          python3 -m pytest ${MODEL_HUB_TESTS_INSTALL_DIR}/pytorch/test_pa_transformation.py -m ${TYPE} --html=${INSTALL_TEST_DIR}/TEST-torch_pagedattention_tests.html --self-contained-html -v --tb=short -n 4
+          python3 -m pytest ${MODEL_HUB_TESTS_INSTALL_DIR}/transformation_tests/test_pa_transformation.py -m precommit --html=${INSTALL_TEST_DIR}/TEST-torch_pagedattention_tests.html --self-contained-html -v --tb=short -n 4
```
Comment (akashchi) [inline .github/workflows/job_pytorch_models_tests.yml:158]:
```suggestion
          python3 -m pytest ${MODEL_HUB_TESTS_INSTALL_DIR}/transformation_tests/test_pa_transformation.py -m ${{ inputs.model_scope }} --html=${INSTALL_TEST_DIR}/TEST-torch_pagedattention_tests.html --self-contained-html -v --tb=short -n 4
```

File: .github/workflows/job_pytorch_models_tests.yml [hunk 3]
```diff
@@ -123,48 +131,47 @@ jobs:
...
 
       - name: StatefulToStateless Test
-        if: always()
+        if: ${{ inputs.model_scope == 'precommit' }}
         run: |
           export PYTHONPATH=${MODEL_HUB_TESTS_INSTALL_DIR}:$PYTHONPATH
-          python3 -m pytest ${MODEL_HUB_TESTS_INSTALL_DIR}/pytorch/test_stateful_to_stateless_transformation.py -m ${TYPE} --html=${INSTALL_TEST_DIR}/TEST-torch_stateful_to_stateless_tests.html --self-contained-html -v --tb=short
+          python3 -m pytest ${MODEL_HUB_TESTS_INSTALL_DIR}/transformation_tests/test_stateful_to_stateless_transformation.py -m ${TYPE} --html=${INSTALL_TEST_DIR}/TEST-torch_stateful_to_stateless_tests.html --self-contained-html -v --tb=short
         env:
           TYPE: ${{ inputs.event == 'schedule' && 'nightly' || 'precommit'}}
```
Comment (akashchi) [inline .github/workflows/job_pytorch_models_tests.yml:169]: There is no `event` input anymore, I guess there should be a change similar to the previous step's.

File: .github/workflows/job_pytorch_models_tests.yml [hunk 4]
```diff
@@ -123,48 +131,47 @@ jobs:
...
           USE_SYSTEM_CACHE: False
-          OP_REPORT_FILE: ${{ env.INSTALL_TEST_DIR }}/TEST-torch_unsupported_ops.log
 
       - name: StatefulToStateless Test
-        if: always()
+        if: ${{ inputs.model_scope == 'precommit' }}
         run: |
           export PYTHONPATH=${MODEL_HUB_TESTS_INSTALL_DIR}:$PYTHONPATH
-          python3 -m pytest ${MODEL_HUB_TESTS_INSTALL_DIR}/pytorch/test_stateful_to_stateless_transformation.py -m ${TYPE} --html=${INSTALL_TEST_DIR}/TEST-torch_stateful_to_stateless_tests.html --self-contained-html -v --tb=short
+          python3 -m pytest ${MODEL_HUB_TESTS_INSTALL_DIR}/transformation_tests/test_stateful_to_stateless_transformation.py -m ${TYPE} --html=${INSTALL_TEST_DIR}/TEST-torch_stateful_to_stateless_tests.html --self-contained-html -v --tb=short
```
Comment (akashchi) [inline .github/workflows/job_pytorch_models_tests.yml:167]:
```suggestion
          python3 -m pytest ${MODEL_HUB_TESTS_INSTALL_DIR}/transformation_tests/test_stateful_to_stateless_transformation.py -m ${{ inputs.model_scope }} --html=${INSTALL_TEST_DIR}/TEST-torch_stateful_to_stateless_tests.html --self-contained-html -v --tb=short
```

File: .github/workflows/linux.yml [hunk 5]
```diff
@@ -557,8 +557,26 @@ jobs:
...
+    name: PyTorch Models tests
+    if: ${{ github.event_name == 'schedule' }}
+    needs: [ Build, Smart_CI, Openvino_tokenizers ]
+    uses: ./.github/workflows/job_pytorch_models_tests.yml
+    with:
+      runner: 'ubuntu-20.04-16-cores'
+      model_scope: 'nightly_scope1'
+
+  PyTorch_Models_Tests_Nightly_Scope2:
+    name: PyTorch Models tests
```
Comment (mvafin) [inline .github/workflows/linux.yml:573]:
```suggestion
    name: PyTorch Models Nightly Scope1 tests
    if: ${{ github.event_name == 'schedule' }}
    needs: [ Build, Smart_CI, Openvino_tokenizers ]
    uses: ./.github/workflows/job_pytorch_models_tests.yml
    with:
      runner: 'ubuntu-20.04-16-cores'
      model_scope: 'nightly_scope1'

  PyTorch_Models_Tests_Nightly_Scope2:
    name: PyTorch Models Nightly Scope2 tests
```

File: .github/workflows/job_pytorch_models_tests.yml (hunk 3 above)
Comment (mvafin) [reply .github/workflows/job_pytorch_models_tests.yml]: removed TYPE

File: .github/workflows/job_pytorch_models_tests.yml (hunk 4 above)
Comment (mvafin) [reply .github/workflows/job_pytorch_models_tests.yml]: Changed to always run with -m precommit

**Review by akashchi — APPROVED** [review]

**Pull request merged**